docker/iroha/keys/notary0@notary.priv

```diff
@@ -0,0 +1 @@
+d7a82f32288ab60d12dc8c1d15110b1712ae99a8a9529d56944deb3267bb219e
```

Comment:
All these files of keys are not needed.

All that we require for tests is single peer with config, genesis.block, admin@test.{priv,pub}, node0.{priv,pub} keys from hyperledger/iroha/examples